src/SDKs/_metadata/azsadmin_resource-manager_backup.txt

```diff
@@ -0,0 +1,5 @@
+﻿Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/backup/readme.md --csharp --csharp-sdks-folder=E:\github\azure-sdk-for-net\src\AzureStack\BackupAdmin\Backup.Admin --version=latest --reflect-api-versions
+
+AutoRest code generation for https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/backup/readme.md failed. Please try again
```

Comment:
looks like code generation failed here, please check in code generated by generate.ps1